custom_components/tahoma/cover.py

```diff
@@ -254,8 +234,8 @@ def supported_features(self):
         if self.has_command(COMMAND_OPEN_SLATS):
             supported_features |= SUPPORT_OPEN_TILT
 
-            if self.has_command(COMMAND_STOP_IDENTIFY, COMMAND_STOP, COMMAND_MY):
-                supported_features |= SUPPORT_STOP_TILT
+        if self.has_command(COMMAND_STOP_IDENTIFY, COMMAND_STOP, COMMAND_MY):
```

Comment:
This one should be nested under the `if` above. Unfortunately we can not differentiate between `SUPPORT_STOP` and `SUPPORT_STOP_TILT` based on the commands, since they are the same.